masking_package/background_functions.py

```diff
@@ -0,0 +1,149 @@
+'''Background functions, functions the user does not have to know about'''
```

Comment:
I have not seen this type of titles before (but if you have then you can ignore this!)
```suggestion
### Background functions, functions the user does not have to know about ###
```

Reply:
I'm doing it like you Dom, for documentation :)